Use custom ExceptionParser when using GoogleAnalyticsTracking

ptrv opened the pull request:
Set custom ExceptionParser in Google Analytics ExceptionReporter in
order to include the stack trace of uncaught exceptions in the tracking
message. This is for Google Analytics v2.

ptrv commented:
Rebased

johnjohndoe commented:
Did you take a look at the [`StandardExceptionParser` class](https://developers.google.com/android/reference/com/google/android/gms/analytics/StandardExceptionParser). It might produce even more detailed information about the crash then with your approach. Please check.

ptrv commented:
Updated. Renamed the variable.

> Did you take a look at the StandardExceptionParser class.

Yes. The whole point of this PR is **not** to use the `StandardExceptionParser`, wich is used in `EasyTracker` -> `ExceptionReporter` and implementing a custom one in order to include more information the stack trace. The `StandardExceptionParser` is providing less information. Look at the source.

johnjohndoe commented:
Thank you!
